FRAuth/FRAuth/Auth/AuthService.swift

```diff
@@ -224,7 +224,7 @@ public class AuthService: NSObject {
     func buildAuthServiceRequest() -> Request {
         
         var header: [String: String] = [:]
-        header[OpenAM.acceptAPIVersion] = OpenAM.apiResource21 + "," + OpenAM.apiProtocol10
```

Comment:
I see it was changed from a previous commit. If you still think it is valid, please add comments about supported APIs in the SDK.

Reply:
Particular change was referenced from old commit; the change has been reverted. Added clarification on API version.